src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/ShopUser.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources
+    xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource class="Sylius\Component\Core\Model\ShopUser">
+        <operations>
+            <operation
+                class="ApiPlatform\Metadata\Get"
+                uriTemplate="/admin/shop-users/{id}"
+                read="false"
+                output="false"
+                controller="ApiPlatform\Action\NotFoundAction"
```

Comment:
Try this:
```suggestion
                class="ApiPlatform\Metadata\NotExposed"
                uriTemplate="/admin/shop-users/{id}"
```
By the way, why do we need this `/shop-users/{id}` URI template? I don't see it used anywhere.